pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -246,6 +255,13 @@ func updateSystemdCgroupInfo(cgroupConfig *libcontainerconfigs.Cgroup, cgroupNam
 
 // Exists checks if all subsystem cgroups already exist
 func (m *cgroupManagerImpl) Exists(name CgroupName) bool {
+	if libcontainercgroups.IsCgroup2UnifiedMode() {
```

Comment:
this eliminates verification on particular controllers being present and available on a cgroupv2 host.

was that intentional?

Reply:
it was intentional as on cgroup v2 the cgroup existence depend only on the cgroup directory being created.  Should I change the test to also verify if the controllers we need are enabled?

Reply:
I've changed the implementation to also check if the controllers we need are enabled.